operations.cc

```diff
@@ -0,0 +1,92 @@
+#include <assert.h>
+#include "operations.h"
+#include "ioperandsmemory.h"
+
+namespace PolishCalcComponent {
+
+value_t add(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a + b);
+}
+
+value_t sub(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a - b);
+}
+
+value_t div(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a / b);
+}
+
+value_t mul(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a * b);
+}
+
+value_t neg(IOperandsMemory* memory){
+    auto a = memory->getOperand();
+
+    memory->putOperand( -a );
+}
+
+enum class Operations: char{
+    ADD = '+',
+    SUB = '-',
+    MUL = '*',
+    DIV = '/',
+    NEG = '!'
```

Comment:
Negative as `!` confusing. `4 !` means `-4` ? In future it will prevent you implementing factorial function